src/utils/date-utils.js

```diff
@@ -8,3 +8,24 @@ const KNOWN_FORMATS = {
 export function dateFormat(date, format) {
   return dayjs(date).format(KNOWN_FORMATS[format] || format)
 }
+
+export function datesDiff(startDate, endDate) {
```

Comment:
There's a somewhat easier way to get this:

```
if (endDate.diff(startDate, 'hour', true) < 24) {
  return endDate.get('date') === startDate.get('date') ? 0 : 1
}
return endDate.diff(startDate, 'day')
```

Reply:
but first we check that the startDate is the same than the endDate, if that condition is false then we check if the hours diff is less than 24 hours then is the next day and if non of the others is true we are returning the days diff

Reply:
The main reason I like the simplification is that this check:

> first we check that the startDate is the same than the endDate

is made redundant by `endDate.diff(startDate, 'hour', true) < 24`. If this is true, it already encompasses the year and month matching, and so we only need to check the day.

Reply:
Actually we've found [`isSame()`](https://day.js.org/docs/en/query/is-same) and should use that :smile: